eessi/testsuite/common_config.py

```diff
@@ -93,10 +94,13 @@ def common_eessi_init(eessi_version=None):
     """
     # Check which EESSI_CVMFS_REPO we are running under
     eessi_cvmfs_repo = os.getenv('EESSI_CVMFS_REPO', None)
+
     if eessi_cvmfs_repo is None:
-        err_msg = "Environment variable 'EESSI_CVMFS_REPO' was not found."
-        err_msg += " Did you initialize the EESSI environment before running the test suite?"
-        raise ValueError(err_msg)
+        warn_msg = "Environment variable 'EESSI_CVMFS_REPO' was not found."
+        warn_msg += " To use EESSI modules, initialize the EESSI environment before running the test suite."
```

Comment:
Maybe we should mention that in the current situation, the local modules will be tested (i.e. maybe we should mention explicitly that not having `EESSI_CVMFS_REPO` set is not per se a problem, as long as that is what the user intended).